ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -392,8 +392,7 @@ message DamlContractState {
   // Optional, if set the contract has been archived.
   google.protobuf.Timestamp archived_at = 2;
 
-  // Optional. The log entry that caused the contract to be archived.
-  DamlLogEntryId archived_by_entry = 3;
+  reserved 3; // was archived_by_entry
```

Comment:
This changes the state value format, does it have consequences? CC @mziolekda

Reply:
It doesn't change it as the field has been optional anyway (it does not need to be populated).